### DISCUSSION TITLE
How to contribute to the project?

### DISCUSSION THREAD

**Discussion by yaten2302** [Leaflet/Leaflet#9093] [Q&A]
Hi everyone,
I want to contribute to this project and also want to be a member of the "Leaflet" community. Can anyone please guide me that how can I contribute to the project, like, from where should I start reading the code?

**Comment by johnd0e** [comment]
You should start from trying to resolve existing issues. And when your PRs get merged, that is called successful contribution.

**Reply by yaten2302** [reply]
Hi, I'll surely try to work on the existing issues, could you also please tell that from where do we have to start reading the code, since it is a very large project.

**Reply by johnd0e** [reply]
https://leafletjs.com/reference.html

**Reply by yaten2302** [reply]
Thank you so much for the api reference.

Sorry for being unclear, actually I want to contribute to this project, and I was trying to ask that how to read and understand the code that is uploaded on [GitHub](https://github.com/Leaflet/Leaflet)
